docs/How to setup mysql plugin.md

```diff
@@ -0,0 +1,9 @@
+# How to setup Mysql plugin for OMS agent
+
+How to enable mySQL plugin to collect mySQL server stats and perf data
+* Install mySQL server on the machine
+* Machine has to have libmysqlclient.so installed(can be downloaded and installed through mysql-devel)
```

Comment:
This is a temp content. Please see the comments/description section of this pull request. We already have MySQL documented for end user that will need to be changed once the omi provider is removed. For now, I am just adding a few lines as readme. Will add more details if you find it useful.